ChangeLog

```diff
@@ -1,3 +1,6 @@
+18.0.0:
+ - Compatibility wiht v11 of the API: https://developers.google.com/google-ads/api/docs/release-notes
```

Comment:
I'd mention the field masks util update and point to the updated guide for Ruby, as this could surprise users who upgrade.